docs/review_process.md

```diff
@@ -21,58 +19,55 @@ so this document intends to describe the internal process in detail.
 
 In general, reviews are driven by the automated [bot](https://github.com/conan-center-bot). The bot is responsible for:
 
-- adding or removing labels (such as [No Beta User](https://github.com/conan-io/conan-center-index/pulls?q=is%3Apr+is%3Aopen+label%3A%22No+Beta+user%22) or [Docs](https://github.com/conan-io/conan-center-index/pulls?q=is%3Apr+is%3Aopen+label%3ADocs))
-- writing comments (most of the time, it's a build status, either failure with logs or success)
-- merging pull requests
-- closing issues (side effect of merging pull requests)
-- starting CI builds
-- assigning CI status (running/failed/successful)
+- Adding or removing labels (such as [No Beta User](https://github.com/conan-io/conan-center-index/pulls?q=is%3Apr+is%3Aopen+label%3A%22No+Beta+user%22) or [Docs](https://github.com/conan-io/conan-center-index/pulls?q=is%3Apr+is%3Aopen+label%3ADocs)).
+- Writing comments (most of the time, it's a build status, either failure with logs or success).
+- Merging pull requests.
+- Closing issues (after merging pull requests with GitHub keywords).
```

Comment:
I was under the impression the GitHub does this automatically 😕

Reply:
yes, this is only side effect of merging PRs. still, it appears in the interface and e-mails like bot did that.